### ISSUE TITLE
Many errors if there are `-ForEach` param

### ISSUE THREAD

**Issue by sergey-s-betke** [pester/vscode-adapter#45]
I see many errors for there test fragment:

````powershell
Describe 'Open Document template' {
	Describe '<Name>' -ForEach @(
		$DestinationTemplateFile | Get-Item |
		ForEach-Object { @{ Name = $_.Name; FullName = $_.FullName } }
	) {
		It 'is valid (by ODFValidator)' {
			{
				java -D"file.encoding=UTF-8" -jar $ODFValidatorJarPath -e -w $FullName
				if ( $LASTEXITCODE -ne 0 )
				{
					Write-Error -Message 'Validation failed!' -ErrorAction 'Stop';
				};
			} | Should -Not -Throw
		}
	}
}
````

All test passed. But i see in console:

````
Starting discovery in 1 files.
Discovery found 1 tests in 243ms.
Running tests.
[-] Describe Open Document template.<Name> failed
 RuntimeException: Unsupported level out output 'c:\Users\sergei.s.betke\Documents\DocTemplates\tests\template\.Tests.ps1'
 в <ScriptBlock>, C:\Users\sergei.s.betke\Documents\WindowsPowerShell\Modules\Pester\5.3.0\Pester.psm1: строка 13191
 в <ScriptBlock>, C:\Users\sergei.s.betke\Documents\WindowsPowerShell\Modules\Pester\5.3.0\Pester.psm1: строка 2007
 в <ScriptBlock>, C:\Users\sergei.s.betke\Documents\WindowsPowerShell\Modules\Pester\5.3.0\Pester.psm1: строка 1949
 в Invoke-ScriptBlock, C:\Users\sergei.s.betke\Documents\WindowsPowerShell\Modules\Pester\5.3.0\Pester.psm1: строка 2113
 в Invoke-TestItem, C:\Users\sergei.s.betke\Documents\WindowsPowerShell\Modules\Pester\5.3.0\Pester.psm1: строка 1225
 в Invoke-Block, C:\Users\sergei.s.betke\Documents\WindowsPowerShell\Modules\Pester\5.3.0\Pester.psm1: строка 826
 в <ScriptBlock>, C:\Users\sergei.s.betke\Documents\WindowsPowerShell\Modules\Pester\5.3.0\Pester.psm1: строка 881
 в <ScriptBlock>, C:\Users\sergei.s.betke\Documents\WindowsPowerShell\Modules\Pester\5.3.0\Pester.psm1: строка 1988
 в <ScriptBlock>, C:\Users\sergei.s.betke\Documents\WindowsPowerShell\Modules\Pester\5.3.0\Pester.psm1: строка 1949
 в Invoke-ScriptBlock, C:\Users\sergei.s.betke\Documents\WindowsPowerShell\Modules\Pester\5.3.0\Pester.psm1: строка 2113
 в Invoke-Block, C:\Users\sergei.s.betke\Documents\WindowsPowerShell\Modules\Pester\5.3.0\Pester.psm1: строка 928
 в <ScriptBlock>, C:\Users\sergei.s.betke\Documents\WindowsPowerShell\Modules\Pester\5.3.0\Pester.psm1: строка 881
 в <ScriptBlock>, C:\Users\sergei.s.betke\Documents\WindowsPowerShell\Modules\Pester\5.3.0\Pester.psm1: строка 1988
 в <ScriptBlock>, C:\Users\sergei.s.betke\Documents\WindowsPowerShell\Modules\Pester\5.3.0\Pester.psm1: строка 1949
 в Invoke-ScriptBlock, C:\Users\sergei.s.betke\Documents\WindowsPowerShell\Modules\Pester\5.3.0\Pester.psm1: строка 2113
 в Invoke-Block, C:\Users\sergei.s.betke\Documents\WindowsPowerShell\Modules\Pester\5.3.0\Pester.psm1: строка 928
 в <ScriptBlock>, C:\Users\sergei.s.betke\Documents\WindowsPowerShell\Modules\Pester\5.3.0\Pester.psm1: строка 881
 в <ScriptBlock>, C:\Users\sergei.s.betke\Documents\WindowsPowerShell\Modules\Pester\5.3.0\Pester.psm1: строка 1988
 в <ScriptBlock>, C:\Users\sergei.s.betke\Documents\WindowsPowerShell\Modules\Pester\5.3.0\Pester.psm1: строка 1949
 в Invoke-ScriptBlock, C:\Users\sergei.s.betke\Documents\WindowsPowerShell\Modules\Pester\5.3.0\Pester.psm1: строка 2113
 в Invoke-Block, C:\Users\sergei.s.betke\Documents\WindowsPowerShell\Modules\Pester\5.3.0\Pester.psm1: строка 928
 в <ScriptBlock>, C:\Users\sergei.s.betke\Documents\WindowsPowerShell\Modules\Pester\5.3.0\Pester.psm1: строка 1662
 в <ScriptBlock>, C:\Users\sergei.s.betke\Documents\WindowsPowerShell\Modules\Pester\5.3.0\Pester.ps1: строка 3
 в <ScriptBlock>, C:\Users\sergei.s.betke\Documents\WindowsPowerShell\Modules\Pester\5.3.0\Pester.psm1: строка 3154
 в Invoke-InNewScriptScope, C:\Users\sergei.s.betke\Documents\WindowsPowerShell\Modules\Pester\5.3.0\Pester.psm1: строка 3161 в Run-Test, C:\Users\sergei.s.betke\Documents\WindowsPowerShell\Modules\Pester\5.3.0\Pester.psm1: строка 1665
 в Invoke-Test, C:\Users\sergei.s.betke\Documents\WindowsPowerShell\Modules\Pester\5.3.0\Pester.psm1: строка 2465
 в Invoke-Pester<End>, C:\Users\sergei.s.betke\Documents\WindowsPowerShell\Modules\Pester\5.3.0\Pester.psm1: строка 5225
 в Invoke-Main, C:\Users\sergei.s.betke\.vscode\extensions\pspester.pester-test-0.0.3\Scripts\PesterInterface.ps1: строка 392 в <ScriptBlock>, C:\Users\sergei.s.betke\.vscode\extensions\pspester.pester-test-0.0.3\Scripts\PesterInterface.ps1: строка 396
 в <ScriptBlock>, <Нет файла>: строка 1
Tests completed in 3.06s
Tests Passed: 1, Failed: 0, Skipped: 0 NotRun: 0
BeforeAll \ AfterAll failed: 1
  - Open Document template.<Name>
PS C:\Users\sergei.s.betke\Documents\DocTemplates>
````

**Comment by JustinGrote** [comment]
Fixed in #23, try installing this VSIX to test:
https://github.com/pester/vscode-adapter/suites/3552882461/artifacts/85185890

**Comment by JustinGrote** [comment]
@sergey-s-betke I just released v2021.8.0 to the marketplace, if you update your extension this should go away.

**Comment by sergey-s-betke** [comment]
ok, thank You!
